File: src/main/java/crd/KMTSpec.java
package crd;

public class KMTSpec {
    private String namespace;
    private String configName;
    private String configKey;

    public String getNamespace() {
        return namespace;
    }

    public void setNamespace(String namespace) {
        this.namespace = namespace;
    }

    public String getConfigName() {
        return configName;
    }

    public void setConfigName(String configName) {
        this.configName = configName;
    }

    public String getConfigKey() {
        return configKey;
    }

    public void setConfigKey(String configKey) {
        this.configKey = configKey;
    }
}

File: src/main/java/crd/KMTSpec.scala
package crd

class KMTSpec {
  private var namespace: String = null
  private var configName: String = null
  private var configKey: String = null

  def getNamespace: String = namespace

  def setNamespace(namespace: String): Unit = {
    this.namespace = namespace
  }

  def getConfigName: String = configName

  def setConfigName(configName: String): Unit = {
    this.configName = configName
  }

  def getConfigKey: String = configKey

  def setConfigKey(configKey: String): Unit = {
    this.configKey = configKey
  }
}
